sport/app/football/conf/context.scala

```diff
@@ -111,9 +111,16 @@ class FootballClient(wsClient: WSClient)(implicit executionContext: ExecutionCon
 
   lazy val apiKey = SportConfiguration.pa.footballKey
 
+  val noneCriticalErrors: List[String] = List(
```

Comment:
```suggestion
  val nonCriticalErrors: List[String] = List(
```

Reply:
Oh we had discussed this yesterday and I had forgotten to apply it. Done now